qiskit/pulse/builder.py

```diff
@@ -1599,13 +1604,14 @@ def shift_phase(phase: float,
         d0 = pulse.DriveChannel(0)
 
         with pulse.build() as pulse_prog:
-            pulse.shift_phase(math.pi, d0)
+            pulse.shift_phase(math.pi, d0, name='pi phase-shift')
```

Comment:
I don't think we need to include this in the examples. In general, the `name` feature for instructions is not often used and the documentation mentions it.